app/components/r/shell/index.js

```diff
@@ -401,6 +404,35 @@ class RShell extends React.Component {
 		return null;
 	}
 
+	// Not sure why this is needed, but it was on the documentation
+	// https://codepen.io/nkbt/pen/eNPoQv?editors=0010
+	onCopyCode = () => {
+		this.setState({
+			copied: true
```

Comment:
As you mentioned, this is not needed. It is likely in the documentation for illustrative purposes. Instead of keeping a `copied` state variable, I suggest we use the `onCopyCode` function to display a notification alerting the user that the code has been successfully copied to the clipboard, e.g. as in the following:

```
session.addNotification({
	title: 'Copied.',
	message: 'Code successfully copied to clipboard',
	level: 'success',
	position: 'tr'
});
```
In addition, I suggest we add a new property to the RShell component to enable / disable the copy button in case people don't want it.